app/_src/production/deployment/multi-zone.md

```diff
@@ -108,7 +108,8 @@ Data plane proxy changes will be propagated within their zones.
 
 ### Zone control plane offline
 
-- New data plane proxies won't be able to join the mesh.
+- New data plane proxies won't be able to join the mesh. This includes new instances (Pod/VM) that are newly created by automatic deployment mechanisms (e.g. a rolling update process), this means a control plane connection failure could block "update by replacement processes" of applications and events that involves new instances to be created.
+- On mTLS enabled meshes, a data plane proxy may fail to refresh its client certificate prior to expiry (defaults to 24h, could be configured at mesh level), thus traffic from/to this data planes will be blocked.
```

Comment:
```suggestion
- On mTLS enabled meshes, a data plane proxy may fail to refresh its client certificate prior to expiry (defaults to 24 hours), thus causing traffic from/to this data planes to fail.
```

Reply:
Thanks, I will also fix other files in a new commit.